src/videojs-http-streaming.js

```diff
@@ -1062,11 +1062,34 @@ class VhsHandler extends Component {
 
     this.player_.tech_.on('keystatuschange', (e) => {
       if (e.status === 'output-restricted') {
-        this.masterPlaylistController_.blacklistCurrentPlaylist({
-          playlist: this.masterPlaylistController_.media(),
-          message: `DRM keystatus changed to ${e.status}. Playlist will fail to play. Check for HDCP content.`,
-          blacklistDuration: Infinity
-        });
+        const masterPlaylist = this.masterPlaylistController_.media();
+        const excludedHDPlaylists = [];
+
+        if (masterPlaylist && masterPlaylist.playlists) {
+          // Assume all HD streams are unplayable and exclude them from ABR selection, then perform
+          // a fastQualityChange to clear the buffer since it may already contain unplayable segments
```

Comment:
Might be better to move the second part of the comment ("...clear the buffer since it may already contain unplayable segments") above the call to `fastQualityChange_`